image_cloudevents/modelmonitor.ipynb

```diff
@@ -0,0 +1,303 @@
+    "        status = data.data.inferenceservice_deployment.parameters.generated.status.state\n",
+    "        inference = data.data.inferenceservice_deployment.parameters.inference\n",
+    "        inference_url = data.data.inferenceservice_deployment.parameters.generated.details.serving.servingurl\n",
+    "        if status == \"RUNNING\":\n",
+    "            break\n",
+    "        print(\"waiting for deployment to be running\")\n",
+    "        time.sleep(api.wait_interval)\n",
+    "    return inference, inference_url"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "api = DkubeApi(URL=os.getenv('DKUBE_URL'),token=os.getenv(\"DKUBE_USER_ACCESS_TOKEN\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Enabling logs"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "while True:\n",
+    "    DEPLOYMENT_ID = api.get_deployment_id(name=MONITOR_NAME)\n",
+    "    if DEPLOYMENT_ID:\n",
+    "        break\n",
+    "    print(\"waiting for deployment to come up\")\n",
+    "    time.sleep(api.wait_interval)\n",
+    "inference, INFERENCE_URL = wait_for_deployment_running(DEPLOYMENT_ID)\n",
+    "if not inference.enable_logs:\n",
+    "    print(\"Enabling logs\")\n",
+    "    serving = DkubeServing(user=DKUBEUSERNAME, name=MONITOR_NAME)\n",
+    "    serving.update_enable_logs(enable_logs=True)\n",
+    "    api.update_inference(serving)\n",
+    "print(\"Inference is up at URL: \", INFERENCE_URL)\n",
+    "%store INFERENCE_URL\n",
+    "%store DEPLOYMENT_ID"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Initializing model monitor"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mm=DkubeModelmonitor(deployemnt_id = DEPLOYMENT_ID)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mm.update_modelmonitor_basics(model_type=ModelType.Classification.value, \n",
+    "                               input_data_type=DataType.Image.value,\n",
+    "                               data_timezone=TimeZone.UTC.value)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mm.update_image_data_shape(height=200, width=200, channel=1)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "DKUBE_BASE_DATASET = \"chest-xray-data\"\n",
+    "training_data = f'{DKUBE_BASE_DATASET}:'+ DKUBEUSERNAME\n",
+    "train_data_version = 'v1:'+api.get_dataset_versions(\n",
+    "    DKUBEUSERNAME,\n",
+    "    DKUBE_BASE_DATASET)[0]['version']['uuid']\n",
```

Comment:
This would be daunting to the users. Can we create a helper function which returns this complicated operation? 
It takes dataset_name, user and version. user and version can be defaulted. non positional parameters

Reply:
okay, I will add it